### DISCUSSION TITLE
Trying to display my api data as a list

### DISCUSSION THREAD

**Discussion by Manuella-Sam** [neuefische/questions#668] [Web]
Hello,

I'm running into issues trying to display my api data in the form of a list as opposed to json Data.

Here is my code :

``` import useSWR from "swr";
import { useRouter } from "next/router";
import { fetcher } from "@/utils/fetcher";
import { v4 as uuidv4 } from "uuid";

const StyledSection = styled.section`
  display: flex;
  justify-content: center;
  flex-direction: column;
  align-items: center;
`;
const StyledHeading = styled.h1`
  display: flex;
  text-align: center;
  font-size: 40px;
  color: darkblue;
`;
const StyledArticle = styled.article`
  border-radius: 20px;
  border: 1px solid lightgrey;
  margin: 5px;
  padding: 10px;
  width: 70%;
  display: flex;
  flex-direction: column;
  align-items: center;
  font-size: 20px;
`;

const StyledSpan = styled.span`
  font-size: 27px;
  color: darkblue;
`;

export default function joblist({ data }) {
  const router = useRouter();
  const { slug: websiteSlug } = router.query;
  const { data } = useSWR(
    `https://arbeitnow.com/api/job-board-api${websiteSlug}`,
    fetcher
  );

  return (
    <>
      <StyledSection>
        <StyledHeading>Job List</StyledHeading>
        {datas.map((data) => {
          return (
            <StyledArticle key={uuidv4()}>
              <StyledSpan>
                <p>{data.title}</p>
              </StyledSpan>

              {data.location}
            </StyledArticle>
          );
        })}
      </StyledSection>
      s
      <StyledButton>
        <StyledLink href="/job-category">Go To Categories</StyledLink>
      </StyledButton>
    </>
  );
}```

**Comment by ThomasF85** [comment]
Hi Manuella,

please provide the following information:
- What kind of issues are you running into? (does the browser crash, do you get error messages, does the page not load at all)
- If you get error messages in the browser or in your vs code terminal, please provide them
- If you have a Vercel link to a preview deployment, please provide it
- If you have a PR with the code snippet, please provide it

**Reply by Manuella-Sam** [reply]
Hi Thomas,

I'm not getting any error messages in the browser or the vs code terminal.I want to display the api data I fetched as a list on the index page.Here is the PR : 
 [https://github.com/Manuella-Sam/dont-forget-to-change-the-project-name/pull/9#issue-1852835680](url)

**Reply by ThomasF85** [reply]
Hi Manuella, the link to your PR is not working.
In your jsx, you say: `{datas.map(...}` but datas is undefined (no variable called `datas` exists).

**Reply by Manuella-Sam** [reply]
https://github.com/Manuella-Sam/dont-forget-to-change-the-project-name/pull/9#issue-1852835680
Oh sorry.Fixed it

**Reply by ThomasF85** [reply]
I left you some comments in the PR. Fix them and you might get your feature to work.
